index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <section id="suzhouhe">
+        <h1>
+          Suzhou River
+        </h1>
+        <img
+          src="https://cdn.glitch.com/cb877aac-d372-492e-b711-f42985913fe5%2F%E4%BC%98%E5%8C%96%E5%90%8E.jpg?v=1602752962003
+      "
+          alt="film poster"
+        />
+        <h3>
+          2000 | romance | 83min | 7.4
+          <a href="https://www.youtube.com/watch?v=3GLEiu-VhvE">
+            <br />
+            watch trailer</a
+          >
+        </h3>
+        
+        <p>
+          The story follows the transient lives of four people at the margins of
+          Chinese soci- ety. An anonymous videographer opens the film with a
+          monologue on the eponymous Suzhou River and love. As the film's narra-
+          tor, the videographer is nevertheless never seen aside from his hands,
+          as the camera serves as his first-person perspective for the audience.
+          He starts by telling the story of a romance between himself and
+          Meimei, a performer at a dive bar in Shanghai...
```

Comment:
It looks like your description copied over some hyphens from the original text, which you should remove here:
```suggestion
          The story follows the transient lives of four people at the margins of
          Chinese society. An anonymous videographer opens the film with a
          monologue on the eponymous Suzhou River and love. As the film's
          narrator, the videographer is nevertheless never seen aside from his hands,
          as the camera serves as his first-person perspective for the audience.
          He starts by telling the story of a romance between himself and
          Meimei, a performer at a dive bar in Shanghai...
```